src/evaluate.py

```diff
@@ -67,6 +68,14 @@ def main():
             metrics_list.append(lang_metrics)
     
     metrics_df = pd.DataFrame(metrics_list)
+
+    results_dir = f"./results/{config.task.task_name}/{config.eval.num_fewshot}/"
+    os.makedirs(results_dir, exist_ok=True)
+    filename = f"{config.model.model_name.split('/')[-1]}.csv"
+
+    metrics_df.to_csv(f'{results_dir}/{filename}', index=False)
+
+    metrics_df = metrics_df.groupby('lang').agg({ metric :'mean' for metric in METRICS }).reset_index()
```

Comment:
What do you think about removing since we already have something that writes the output to Wandb in lines 105 & 106?

Here's a screenshot of what it looks like in wandb and you can download it as a csv if needed:

![image](https://github.com/user-attachments/assets/6fd38639-243d-468e-99f6-a020ee3b6df8)